src/NServiceBus.Core/obsoletes.cs

```diff
@@ -1178,7 +1178,7 @@ public class MsmqDequeueStrategy
     }
 }
 
-namespace NServiceBus.Unicast.Transport
+namespace NServiceBus.Unicast.Transports
```

Comment:
> as part of this PR we changed it to Transport. This commit reverts it back where necessary to Transports.

maybe the diff isn't showing it properly, but on the PR diff this is still shown as change, so it would be `NServiceBus.Unicast.Transport` on the release branch? Ignore that in case it's just a diff issue and if you're sure you got it right ;)